<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gesDepart</class>
 <widget class="QDialog" name="gesDepart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des d√©parts</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res.qrc">
    <normaloff>:/icones/Icones/depart.jpg</normaloff>:/icones/Icones/depart.jpg</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="AgentEdit">
             <property name="placeholderText">
              <string>Agent</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="typeEdit">
             <property name="placeholderText">
              <string>Type </string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="departTableView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btNouveau">
           <property name="text">
            <string>Nouveau</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/add.ico</normaloff>:/icones/Icones/add.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btModifier">
           <property name="text">
            <string>Modifier</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/05-Edit-icon.png</normaloff>:/icones/Icones/05-Edit-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btSupprimer">
           <property name="text">
            <string>Supprimer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btType">
           <property name="text">
            <string>Type Depart</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/10-Search-icon.png</normaloff>:/icones/Icones/10-Search-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btFermer">
         <property name="text">
          <string>Fermer</string>
         </property>
         <property name="icon">
          <iconset resource="../Res.qrc">
           <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>gesDepart</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
